src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -106,3 +106,124 @@ def update_epoch(self, epoch):
             self.now_epoch = epoch
             for k in self.if_init_list:
                 self.if_init_list[k] = True
+
+class FPGMPruner(Pruner):
+    """
+    A filter pruner via geometric median.
+    "Filter Pruning via Geometric Median for Deep Convolutional Neural Networks Acceleration",
+    https://arxiv.org/pdf/1811.00250.pdf
+    """
+
+    def __init__(self, model, config_list):
+        """
+        Parameters
+        ----------
+        model : pytorch model
+            the model user wants to compress
+        config_list: list
+            support key for each list item:
+                - pruning_rate: percentage of convolutional filters to be pruned.
+        """
+        super().__init__(model, config_list)
+        self.mask_list = {}
```

Comment:
`mask_list` becomes `mask_dict`, and is defined in the Pruner class

Reply:
Thanks, fixed.